app/code/core/Mage/Adminhtml/Block/Widget/Form.php

```diff
@@ -197,7 +197,7 @@ protected function _setFieldset($attributes, $fieldset, $exclude=array())
                         'label'     => $attribute->getFrontend()->getLabel(),
                         'class'     => $attribute->getFrontend()->getClass(),
                         'required'  => $attribute->getIsRequired(),
-                        'note'      => $this->escapeHtml($attribute->getNote()),
+	                  'note'      => $this->escapeHtml($attribute->getNote()),
```

Comment:
I'm sorry, what I meant was leave Form.php unchanged so that it doesn't show up in the list of files that were changed. But I'm not sure how a file can be "uncommitted". May be something like in this [stackoverflow](https://stackoverflow.com/questions/39459467/remove-a-modified-file-from-pull-request).

Reply:
Done!